File: LeetCode/Python/#412 Fizz Buzz.py
def fizzBuzz(n):
    res = []
    for i in range(1,n+1):
        res.append("Fizz"*(not i%3)+"Buzz"*(not i%5)+str(i)*((i%3)!=0 and (i%5)!=0))
    print(res)
    return res

fizzBuzz(15)

File: LeetCode/JavaScript/#412 Fizz Buzz.js
var fizzBuzz = function(n) {
    var res = ["1"];
    var i ;
    for(i = 2; i <= n;i++){
    	if(i%3===0){
    		if(i%5===0){
    			res.push("FizzBuzz");
    			continue;
    		}
    		res.push("Fizz");
    	}else{
    		if(i%5===0){
    			res.push("Buzz");
    		}else{
    			res.push(""+i);
    		}
    	}
    }
    return res;
};

fizzBuzz(100);